src/components/IPFS/Upload.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react'
+import Dropzone from './Dropzone'
+import styled from '@emotion/styled'
+import ipfsClient from 'ipfs-http-client'
+import { loggedIn, getToken } from './Auth'
+import { getConfig, getDev } from './Config'
+import Loader from '../Loader'
+import mq from 'mediaQuery'
+
+const Container = styled('div')`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  align-items: center;
+  text-align: left;
+  overflow: hidden;
+  ${mq.small`
+    text-align: center;
+    margin-top: 10px;
+  `}
+`
+
+const FileName = styled('span')`
+  margin-bottom: 0px;
+  font-size: 16px;
+
+  ${mq.small`
+    font-size: 20px;
+    margin-bottom: 12px;
+  `}
+`
+
+const ErrorMsg = styled('span')`
+  margin-bottom: 0px;
+  font-size: 16px;
+  color: red;
+  ${mq.small`
+    font-size: 20px;
+    margin-bottom: 8px;
+  `}
+`
+
+const Files = styled('div')`
+  align-items: flex-start;
+  justify-items: flex-start;
+`
+
+const Upload = props => {
+  const [files, setFiles] = useState([])
+  const [upload, setUpload] = useState(false)
+  const [uploadError, setUploadError] = useState(false)
+
+  const sendRequest = newfiles => {
```

Comment:
This function will keep getting created each render. Please refactor this using [`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback) so it doesn't get created every time.